packaging/fetch_requirements

```diff
@@ -323,6 +327,8 @@ def file_downloader(download_queue, log_lock, has_errors):
             has_errors.set()
             with log_lock:
                 print('Error downloading {url}: {e}'.format(url=source, e=e))
+                raise RuntimeError('Error downloading {url}: {e}'
```

Comment:
I thing it'll be good to add it permanently here, otherwise the CI is mute about which links actually work, since the print goes to a pod, which is not easily accessible